molecule/molecule-runtime/src/main/java/app/cash/molecule/molecule.kt

```diff
@@ -105,6 +105,7 @@ fun <T> CoroutineScope.launchMolecule(
     }
   }
   coroutineContext.job.invokeOnCompletion {
+    composition.dispose()
```

Comment:
Not sure if there's any significance to the ordering here...

Reply:
It's all single-threaded so order doesn't matter.